src/main/java/com/openshift/internal/restclient/URLBuilder.java

```diff
@@ -105,7 +104,7 @@ public URLBuilder resource(IResource resource) {
 	}
 
 	public URLBuilder addParmeter(String key, String value) {
-		params.put(key, value);
+		params.add(new AbstractMap.SimpleEntry<>( key, value ));
```

Comment:
Haven't done any test work yet. Wanted to ensure you were satisfied with the direction.